brains/andalite.py

```diff
@@ -1,32 +1,38 @@
 import ast
 import asyncio
 import pprint
+import pygame
 import queue
 import threading
 import websockets
 
 from sapiens import Sapiens
 from sync import game_sync_classes
-
+import visualization.textlog
 
 class Andalite(Sapiens):
     name = 'andalite'
     lastSent = {}
 
-    def __init__(self, game, *_, server='ws://localhost:8765', **kwargs):
+    def __init__(self, game, server='ws://localhost:8765', *_, **kwargs):
         Sapiens.__init__(self, game, **kwargs)
         self.telepathy = Telepathy(server)
         # get sync class for the currently loaded game.
         game_sync_class = game_sync_classes.get_game_sync_class(game.emu.gameinfo['name'])
         self.game_sync = None
         if game_sync_class is not None:
             self.game_sync = game_sync_class(self.game.PeekMemoryRegion, self.game.PokeMemoryRegion)
+        visualization.textlog.init_text_log_surface(self.game.ScreenSize())
```

Comment:
so the text log ends up the same width as the game screen? might make more sense to just have a fixed width for it instead